lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Model for Course Card.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.set({
+                        name: data.display_name,
+                        courseStart: data.course_start,
```

Comment:
We do need a course card model as part of this PR, that being how the data is being added to the page. I just don't think we should be renaming the keys to camelCase.